backend/src/auth/auth.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+import { PassportModule } from '@nestjs/passport';
+import { JwtModule } from '@nestjs/jwt';
+import { JWT_EXPIRE } from 'src/constants';
+import { AuthService } from './auth.service';
+import { AuthResolver } from './auth.resolver';
+import { JwtStrategy } from './jwt.strategy';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Module({
+  imports: [
+    PassportModule.register({ defaultStrategy: 'jwt' }),
+    JwtModule.register({
+      secret: process.env.JWT_SECRET,
+      signOptions: { expiresIn: JWT_EXPIRE },
+    }),
+  ],
+  providers: [
+    AuthService,
+    AuthResolver,
+    JwtStrategy,
+    // GqlAuthGuard,
```

Comment:
Is this still going to be used?

Reply:
Nah, you can directly import it from guards. I'll remove it